src/intro.md

```diff
@@ -1,13 +1,11 @@
 # Introduction
 
-Yorick is a fork of the Rust programming language which aims to build a
-meta-tracing system using Rust as a meta-language. Imagine PyPy, but where your
-interpreter is written in Rust instead of RPython.
+yk is a meta-tracing system built on LLVM.
 
 Note that the system is in early stages of development: don't expect it to work
 yet.
 
-Yorick is developed by [The Software Development Team](https://soft-dev.org/)
+yk is developed by [The Software Development Team](https://soft-dev.org/)
```

Comment:
I think we should delete this sentence: we're the initial developers, but we don't want people to feel they can't contribute. Thoughts?

Reply:
OK by me.

Reply:
OK, off with its head :)